code/modules/mob/living/brain/brain_item.dm

```diff
@@ -37,7 +37,7 @@
 		if(C.has_brain_worms())
 			var/mob/living/simple_animal/borer/B = C.has_brain_worms()
 			B.leave_victim() //Should remove borer if the brain is removed - RR
-		transfer_identity(C)
+	transfer_identity(C)
```

Comment:
pretty sure the ```special``` is how changelings get away with dropping fake brains without BECOMING those brains.